zulip_botserver/zulip_botserver/server.py

```diff
@@ -178,8 +212,16 @@ def handle_bot() -> str:
 def main() -> None:
     options = parse_args()
     global bots_config
+
     try:
-        bots_config = read_config_file(options.config_file, options.bot_name)
+        if options.use_env_vars:
+            bots_config = read_config_from_env_vars(options.bot_name)
+        elif options.config_file:
+            bots_config = read_config_file(options.config_file, options.bot_name)
+        else:
+            sys.exit("""Configuration Error: Please pass --use-env-vars or -e along with properly formatted environment varibales
+                     Or pass the path to a properly formatted config file with --config-file path/to/config/file""")
```

Comment:
We can do this a little more cleanly by making the `--config-file` and `--use-env-vars` options [be mutually exclusive and required](https://docs.python.org/3/library/argparse.html#mutual-exclusion).

Reply:
We can't hit this `else` case anymore, because the mutually-exclusive-but-required options we now configured above.